lib/boilerplate.js

```diff
@@ -62,6 +62,72 @@ module.exports = function (moduleName, options = {}) {
           };
         `
       }
+    },
+    a3: {
+      'create-module': {
+        moduleConfig: stripIndent`
+          module.exports = {
+            extend: '@apostrophecms/module',
+            init(self) {
+
+            }
+          };
+        `
+      },
+      'create-piece': {
+        pieceConfig: stripIndent`
+          module.exports = {
+            extend: '@apostrophecms/piece-type',
+            options: {
+              label: '${util.titleCase(moduleName.replace(/-/g, ' '))}',
+              // TODO: Update the plural label for editors
+              pluralLabel: '${util.titleCase(moduleName.replace(/-/g, ' '))}',
+            },
+            fields: {
+              add: {},
+              group: {}
+            }
+          };
+        `,
+        pagesConfig: stripIndent`
+          module.exports = {
+            extend: '@apostrophecms/piece-page-type',
+            options: {
+              label: '${util.titleCase(moduleName.replace(/-/g, ' '))} Page',
+              pluralLabel: '${util.titleCase(moduleName.replace(/-/g, ' '))} Pages',
+            },
+            fields: {
+              add: {},
+              group: {}
+            }
+          };
+        `
+      },
+      'create-widget': {
+        widgetsConfig: stripIndent`
+          module.exports = {
+            extend: '@apostrophecms/widget-type',
+            options: {
+              label: '${util.titleCase(moduleName.replace(/-/g, ' '))} Widget',
```

Comment:
Think you should calculate this often-used string up front before the interpolation blocks.